src/utils/helpers.js

```diff
@@ -0,0 +1,34 @@
+import { disableButton, resetValidation, settings, toggleButtonState } from "../scripts/validation";
+
+export function renderLoading(isLoading, button, defaultText='Save' , loadingText="Saving. . .") {
+  if (isLoading) {
+    button.textContent = loadingText;
+  } else {
+    button.textContent = defaultText;
+  }
+}
+
+export function handleSubmit(request, evt, loadingText = 'Saving...', inactiveButtonClass) {
+  evt.preventDefault();
+
+  const submitButton = evt.submitter;
+
+  if (!submitButton) return;
+
+  const initialText = submitButton.textContent;
+  renderLoading(true, submitButton, initialText, loadingText);
+  request()
+    .then(() => {
+      evt.target.reset();
+      disableButton(submitButton, inactiveButtonClass)
```

Comment:
* I can't delete 2 cards in a row. The button gets **disabled**


Please, check if you have passed `inactiveButtonClass`


```js

    .then(() => {
      evt.target.reset();

      if (inactiveButtonClass) {   // if inactiveButtonClass is passed, then disable the button
        disableButton(submitButton, inactiveButtonClass);  
      }
    })
    .catch(console.error)
```